src/Core/Tool/Authorizer/CSVAuthorizer.php

```diff
@@ -55,7 +50,7 @@ public function isAllowed(Entity $entity, $privilege)
 		$user = $this->getUser();
 
 		// Allow role with the right permissions
-		if ($this->hasPermission($user)) {
+		if ($this->acl->hasPermission($user, Permission::DataImport)) {
```

Comment:
I think this ends up clearer. We can see that the permission check is really happening in the Acl, not this class